### DISCUSSION TITLE
Marketing strategy

### DISCUSSION THREAD

**Discussion by thienphuoc** [axmolengine/axmol#1385] [General]
The current engine seems quite stable to me, but I think we should have a marketing strategy, right ? 👍 . For example, creating mini games, ui tools, writing documents, rewriting books, etc., because its structure is very similar to Cocos2d-x, so it won't take too much time! Currently, no one knows what Axmol is 😩

**Comment by iAndyHD3** [comment]
we need an axmol website where the project is presented in a more fancy way. and include links to wasm tests

**Reply by aismann** [reply]
And who want do it?
One step is done:  Please add Axmol to the runtime list of FairyGUI: #1359

**Reply by thienphuoc** [reply]
I think we just need to create mini games as tutorials for newbies, so they find it easy to get started. 

This cannot be accomplished by one person but requires all of us. 

I have many games written in Cocos2d-x, and I am planning to port them to Axmol and introduce them widely!

**Reply by iAndyHD3** [reply]
yea i agree with this. at the moment there are only tests for reference, it would be cool to have some official example games

**Comment by aismann** [comment]
A good idea is to port [16-Games](https://github.com/embeddedmz/16-Games) to Axmol.
YouTube: [Let's make 16 games in C++/SFML! by FamTrinli](https://www.youtube.com/channel/UCC7qpnId5RIQruKDJOt2exw)
